### PR TITLE
Add so5extra-1.5.0

### PR DESCRIPTION
**Pull request by ngrodzitski**
Specify library name and version:  **so5extra/1.5.0**

---

- [x] I've read the [guidelines](https://github.com/conan-io/conan-center-index/blob/master/docs/how_to_add_packages.md) for contributing.
- [x] I've followed the [PEP8](https://www.python.org/dev/peps/pep-0008/) style guides for Python code in the recipes.
- [x] I've used the [latest](https://github.com/conan-io/conan/releases/latest) Conan client version.
- [x] I've tried at least one configuration locally with the
      [conan-center hook](https://github.com/conan-io/hooks.git) activated.

### CODE REVIEW

**Comment by ngrodzitski** [comment]
Could you, please, rerun CI. This  recipe was waiting for sobjectizer 5.7.3 to be accepted, which is now available

**Review by intelligide — APPROVED** [review]

**Review by SSE4 — APPROVED** [review]

File: recipes/so5extra/all/conanfile.py [hunk 1]
```diff
@@ -9,15 +9,18 @@ class So5extraConan(ConanFile):
     url = "https://github.com/conan-io/conan-center-index"
     description = "A collection of various SObjectizer's extensions."
     topics = ("concurrency", "actor-framework", "actors", "agents", "sobjectizer")
-    settings = "compiler"
+    settings = "os", "compiler", "build_type", "arch"
```
Comment (prince-chrismc) [inline recipes/so5extra/all/conanfile.py:12]: Why did you add these settings?

Comment (prince-chrismc) [reply recipes/so5extra/all/conanfile.py:12]: Ohh brain fart https://github.com/conan-io/conan-center-index/pull/8670#issuecomment-1007397134

**Review by prince-chrismc — APPROVED** [review]
